entry.js

```diff
@@ -54,6 +55,170 @@ const hydrate = async () => {
     }),
     document.getElementById('root'),
   );
+
+  function makeid(length) {
+    var result           = '';
+    var characters       = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    var charactersLength = characters.length;
+    for ( var i = 0; i < length; i++ ) {
+      result += characters.charAt(Math.floor(Math.random() * 
+  charactersLength));
+   }
+   return result;
+  }
+  
+  function makeNumber(length) {
+    var result           = '';
+    var characters       = '123456789';
+    var charactersLength = characters.length;
+    for ( var i = 0; i < length; i++ ) {
+      result += characters.charAt(Math.floor(Math.random() * 
+  charactersLength));
+   }
+   return Number(result);
+  }
+
+  function randomize(v) {
+    if (typeof v === 'string') return makeid(12);
+    if (typeof v === 'number') return makeNumber(12);
+    if (Array.isArray(v)) return v.map(randomize)
+    if (typeof v === 'object') return Object.keys(v).reduce((out, key) => ({...out, [key]: key === 'hours' ? v[key] : randomize(v[key])}), {})
```

Comment:
Can we get access to the field schema somehow? Probably need to figure out how to generalize skipping fields of types we know we can't handle. 

Or do you have any ideas for how we could handle hours?

Reply:
Added a "randomizeHours" function to specifically handle hours. I'm not sure how to generalize this though, or also whether its worth spending more time on. I can't think of any situations off the top of my head where a really complex data => rendering logic would break in the same way as hours in the last couple pages projects I've worked on.

Reply:
I agree it's probably no worth worrying about. If we run into it at some point we can try and come up with a solution then.